### PR TITLE
update phase command to upload certs after expiration

### LINKED ISSUE #17453: Issue with kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/
**This is a Bug Report**
More of an enhancement than a bug report tbh

**Problem:**
As discussed with @neolit123  in kubernetes/kubeadm [1886](https://github.com/kubernetes/kubeadm/issues/1886):  
Running the
`kubeadm join my-url-com:5443 --token <token> --discovery-token-ca-cert-hash sha256:<hash> --control-plane --certificate-key <new-token>`  
returns the following error:
`error execution phase control-plane-prepare/download-certs: error downloading certs: the Secret does not include the required certificate or key - name: external-etcd-ca.crt, path: /etc/kubernetes/pki/etcd/ca.crt`
After discussion it turns out that we need to use the `--config` flag with the phase command that uploads certs to the external etcd, as follows: 
`kubeadm init phase upload-certs --upload-certs --config kubeadm-config.yaml`
**Proposed Solution:**
Update the documentation to show the need to use the `--config` flag with an appropriate yaml config file.

**Page to Update:**
https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#uploading-control-plane-certificates-to-the-cluster

**Pull Request:** #17455

### PR DESCRIPTION
**Pull request by Joseph94m**
Fixes #17453

### CODE REVIEW

**Comment by Joseph94m** [comment]
@neolit123

**Comment by neolit123** [comment]
sorry for the CLA troubles.
/lgtm
/sig-cluster-lifecycle

**Comment by neolit123** [comment]
> Issue #17453

if you change that to Fixes #17453 it will auto close the related issue.

**Comment by Joseph94m** [comment]
> sorry for the CLA troubles.
> /lgtm
> /sig-cluster-lifecycle

Don't worry about it. Next time it will be much smoother :)

**Comment by neolit123** [comment]
@Joseph94m 
it seems you picked dev-1.17 as the target branch and it should have been for master.
TL;DR
master - generic changes that apply for old release too
dev-xx - new features that are added this k8s cycle

changing the branch from the github ui will probably not work so great, so you might have submit a new PR again...

**Comment by Joseph94m** [comment]
yeah it won't work so great because i forked from dev-1.17. I will fork from master and recommit the changes. and open another PR
